### PR TITLE
edited python script to seed test user into database

### PR DESCRIPTION
**Pull request by GemTay**
## Purpose
Seeds a test user account into the database to save dev's having to create a new account each time they bring up the containers 

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/seeding/dynamodb.py [hunk 1]
```diff
@@ -113,3 +116,25 @@
...
     #print(json.dumps(response['Item'], indent=4, separators=(',', ': ')))
+
+# test user details
+
+actorUsersTable = dynamodb.Table(os.environ['DYNAMODB_TABLE_ACTOR_USERS'])
+
+actorUsers = [
+    {
+        'Id': 'bf9e7e77-f283-49c6-a79c-65d5d309ef77',
+        'Email': 'test.user@digital.justice.gov.uk',
```
Comment (andrewpearce-digital) [inline service-api/seeding/dynamodb.py:127]: I'd recommend using an email address we can get to if we ever want to be able to unlock an account on preprod etc.

I've been using something similar to opg-use-an-lpa+test-user@digital.justice.gov.uk. That's an email address we can get to if we need.

File: service-api/seeding/dynamodb.py [hunk 2]
```diff
@@ -113,3 +116,25 @@
         Key={'ActorCode': i['ActorCode']}
     )
     #print(json.dumps(response['Item'], indent=4, separators=(',', ': ')))
```
Comment (andrewpearce-digital) [inline service-api/seeding/dynamodb.py:118]: should this still be commented out?

File: service-api/seeding/dynamodb.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import os
 import boto3
 import json
+import passlib
```
Comment (andrewpearce-digital) [inline service-api/seeding/dynamodb.py:4]: I think this line is redundant now that you're importing sha256 directly, and not calling passlib itself anywhere

File: service-api/seeding/dynamodb.py (hunk 2 above)
Comment (GemTay) [reply service-api/seeding/dynamodb.py:118]: When uncommented it produces this error: 
```
TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Decimal is not JSON serializable
```
Just looking into it, I'm guessing it's the format of the expiry date thats causing the error... unsure just yet though

File: service-api/seeding/dynamodb.py (hunk 1 above)
Comment (cooperaj) [reply service-api/seeding/dynamodb.py:127]: I'll just quietly remove my review... _hides_

**Comment by GemTay** [comment]
Just tested login in the UAT env - it works! Horray!

**Pull request merged**